docs/json_browser/markdowns/ega-2-definitions-check-that-the-object_ids-accession-pattern-and-object_type-match-anyof-individual-object_id-and-object_type-check-properties-object_id-properties-pattern-of-an-ega-individuals-id-egai.md

```diff
@@ -0,0 +1,31 @@
+# Pattern of an EGA Individual's ID (EGAI...) Schema
+
+```txt
+https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/object-id-and-object-type-check/anyOf/13/properties/object_id/properties/ega_accession
+```
+
+
+
+| Abstract            | Extensible | Status         | Identifiable            | Custom Properties | Additional Properties | Access Restrictions | Defined In                                                                                |
+| :------------------ | :--------- | :------------- | :---------------------- | :---------------- | :-------------------- | :------------------ | :---------------------------------------------------------------------------------------- |
+| Can be instantiated | No         | Unknown status | Unknown identifiability | Forbidden         | Allowed               | none                | [EGA.common-definitions.json*](../out/EGA.common-definitions.json "open original schema") |
+
+## ega_accession Type
+
+`string` ([Pattern of an EGA Individual's ID (EGAI...)](ega-2-definitions-check-that-the-object_ids-accession-pattern-and-object_type-match-anyof-individual-object_id-and-object_type-check-properties-object_id-properties-pattern-of-an-ega-individuals-id-egai.md))
+
+## ega_accession Constraints
+
+**pattern**: the string must match the following regular expression: 
+
+```regexp
+^EGAI[0-9]{11}$
```

Comment:
Validated with [Regex101](https://regex101.com/) . The pattern is correct and matches the EGAI 😊